c/planarityCommandLine.c

```diff
@@ -58,6 +59,9 @@ int commandLine(int argc, char *argv[])
 	else if (strncmp(argv[1], "-t", 2) == 0)
 		Result = callTestGraphFunctionality(argc, argv);
 
+	else if (strcmp(argv[1], "-m") == 0)

```

Comment:
This can be deleted because it is adding a feature that we kind of already via use  of the command line. It'd be nicer to add the feature (along with another related one) at a different time. When this is added, some update of the help message, the man page, etc. has to happen.